### PR TITLE
Fix selenium test flakiness

### PR DESCRIPTION
**Pull request by martinmladenov**

### CODE REVIEW

**Review by mauricioaniche — APPROVED** [review]
LGTM

"Status done" seems the right thing to wait for!

File: src/test/java/selenium/pageobjects/BasePageObject.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        wait.until(new ExpectedCondition<Boolean>() {
+            public Boolean apply(WebDriver webDriver) {
+                try {
+                    return until.get();
+                } catch (Exception ex) {
+                    return false;
+                }
+            }
+        });
+    }
```
Comment (martinmladenov) [inline src/test/java/selenium/pageobjects/BasePageObject.java:42]: Is this the best way of preventing code duplication here?

Comment (Arraying) [reply src/test/java/selenium/pageobjects/BasePageObject.java:42]: I can't think of a better solution off the top of my head.

Comment (mauricioaniche) [reply src/test/java/selenium/pageobjects/BasePageObject.java:42]: Also looks good to me

**Pull request merged**